### PR TITLE
Fix #968 : Reenable all landscape tests in app module

### LINKED ISSUE #968: Reenable all landscape tests in app module
In app module there are test cases which are getting ignored right now because of landscape feature. Now we can reenable all these test cases. You can find these test cases by this line in various files:
`@Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.`

Note: You have to focus on `app` module tests only.

### PR DESCRIPTION
**Pull request by SayantanBanerjee16**
## Explanation

Fixes #968 :

There are four files where Landscape tests are still ignored:- 
```
- AudioFragmentTest
- StateFragmentTest
- InputInteractionViewTestActivityTest
- ConceptCardFragmentTest
```

The first 3 still don't have any landscape mode so far and so I had kept them un-affected. The last one landscape mode enabled and tests passed.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/topic/conceptcard/ConceptCardFragmentTest.kt [hunk 1]
```diff
@@ -98,13 +98,9 @@ class ConceptCardFragmentTest {
   }
 
   @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/topic/conceptcard/ConceptCardFragmentTest.kt:101]: There are some unused imports. Please remove them

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, one nit suggested.

File: app/src/sharedTest/java/org/oppia/app/topic/conceptcard/ConceptCardFragmentTest.kt (hunk 1 above)
Comment (SayantanBanerjee16) [reply app/src/sharedTest/java/org/oppia/app/topic/conceptcard/ConceptCardFragmentTest.kt:101]: Done.

**Comment by SayantanBanerjee16** [comment]
@rt4914 Done with the nit update. Do merge. Thanks :)

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**